main.go

```diff
@@ -4,13 +4,35 @@ Copyright © 2022 Netmaker Team <info@netmaker.io>
 package main
 
 import (
+	"log"
+	"strings"
+
+	"github.com/blang/semver"
 	"github.com/gravitl/netclient/cmd"
 	"github.com/gravitl/netclient/config"
+	"github.com/rhysd/go-github-selfupdate/selfupdate"
 )
 
 var version = "dev"
 
+func autoUpdate() {
```

Comment:
Let's put this in a different package, then import to main so that the GUI can later on use the it

Reply:
done